### PR TITLE
adding Versions section to readme

### PR DESCRIPTION
**Pull request by SomkaPe**
Adding Versions section with current, minimum  recommended versions, link  to ChangeLog

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,11 @@ and with [Microsoft Active Directory Federation Services](http://en.wikipedia.or
 ## Security Vulnerability in Versions < 1.4.6 and 2.0.0
 _passport-azure-ad_ has a known security vulnerability affecting versions <1.4.6 and 2.0.0. Please update to >=1.4.6 or >=2.0.1 immediately. For more details, see the [security notice](https://github.com/AzureAD/passport-azure-ad/blob/master/SECURITY-NOTICE.MD).
 
+## Versions
+Current version - 2.0.1  
+Minimum  recommended version - 1.4.6  
+[Change log](https://github.com/AzureAD/passport-azure-ad/blob/master/CHANGELOG.md)
```
Comment (polita) [inline README.md]: Please put a little more prose here. I recommend something like "You can find the changes for each version in the [change log]" .

Comment (SomkaPe) [inline README.md]: actually some projects (Wilson and Passport) describe changes only for current version, 
is it fine to have different "Versions" template ?

Comment (polita) [inline README.md]: Yes, that's fine. Going forward, we'll add a changelog entry for each release for Wilson and Passport.

**Comment by polita** [comment]
:shipit:

**Comment by polita** [comment]
Please also merge into release1x and master.

**Pull request merged**